lib/deps.ts

```diff
@@ -220,21 +220,21 @@ function loadModulesInternal(root, rootDepType, parent, options?): Promise<Packa
 
         return modules;
       });
-    }).then(function (modules) {
-      let deps = Object.keys(modules.dependencies);
+    }).then(function (mods) {
```

Comment:
As the above are going from abbreviations -> non abbreviated can we leave this as `modules` for consistency purposes?

Reply:
This is considered to be a shadow variable, because in the .then above we already have `modules` and we return modules. 

Then in this function we expect to receive `modules` so it is triggering the rule.

I took a look at making this async await to get rid of the problem, but it is going to take quite a bit of effort this code is 🍝 😓